WebDriverAgentLib/Utilities/FBAlertsMonitor.m

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBAlertsMonitor.h"
+
+#import "FBAlert.h"
+#import "XCUIApplication.h"
+#import "XCUIApplication+FBAlert.h"
+
+static const NSTimeInterval FB_MONTORING_INTERVAL = 2.0;
+
+@interface FBAlertsMonitor()
+
+@property (atomic) BOOL isMonitoring;
+
+@end
+
+@implementation FBAlertsMonitor
+
+- (instancetype)init
+{
+  if ((self = [super init])) {
```

Comment:
`(self = [super init])`?

Reply:
yep, what's wrong with it?

Reply:
I do put brackets to emphasise assigning a variable inside if